source/whats-new.txt

```diff
@@ -19,13 +19,24 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.11 <version-4.11>`
 * :ref:`Version 4.10 <version-4.10>`
 * :ref:`Version 4.9 <version-4.9>`
 * :ref:`Version 4.8 <version-4.8>`
 * :ref:`Version 4.7 <version-4.7>`
 
 .. _upcoming-breaking-changes:
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+The {+driver-short+} v4.11 release includes the following new features:
+
+- Adds support for `free-threaded CPython <https://docs.python.org/3/howto/free-threading-python.html>`__
+  when running Python v3.13+. 
```

Comment:
S: instead of hyperlinking the text, I'd add a brief description of where this link is going
```suggestion
- Adds support for free-threaded CPython when running Python v3.13+. For more information about free threading, see the `Python documentation <https://docs.python.org/3/howto/free-threading-python.html>`__.
```